### ISSUE TITLE
Any plan to support glusterfs 7?

### ISSUE THREAD

**Issue by adimate90** [gluster/gstatus#17]
Hello,

Today I upgraded glusterfs to 7.0 ( yay ).
However, after the upgrade I noticed that gstatus is not working any more :(
Is there any plan to make a release to gstatus in order to support gluster 7?

root@demo2:~# gluster volume status
Status of volume: gfs
Gluster process                             TCP Port  RDMA Port  Online  Pid
------------------------------------------------------------------------------
Brick demo1:/gluster/bricks/1/brick         49152     0          Y       1846
Brick demo2:/gluster/bricks/2/brick         49153     0          Y       24646
Brick arbiter:/gluster/bricks/3/brick       49152     0          Y       16762
Self-heal Daemon on localhost               N/A       N/A        Y       26784
Self-heal Daemon on demo1                   N/A       N/A        Y       1931
Self-heal Daemon on arbiter                 N/A       N/A        Y       7603

Task Status of Volume gfs
------------------------------------------------------------------------------
There are no active volume tasks

root@demo2:~# gstatus -a
gstatus is not compatible with this version of glusterfs 7.0

Thanks!

**Comment by Apollon77** [comment]
if it is just that to avoid the error update the script as described here https://github.com/gluster/gstatus/issues/11#issuecomment-484870714 ...

**Comment by adimate90** [comment]
So, thanks to @Apollon77  and @markebbers's [comment](https://github.com/gluster/gstatus/issues/11#issuecomment-484870714), I managed to get gstatus working with 7.0.

What I did:
1. I edited  ./gstatus/gstatuscfg/config.py file and set `min_version =  7.0`
2. The I ran: `python setup.py install` 

P.S. This does not guarantee by any means that all the features of gstatus are working correctly, but for my scenario, where I use gstatus to monitor the cluster via zabbix, it does the job.
